### PR TITLE
Added Test Framework

### PR DESCRIPTION
**Pull request by Fleshgrinder**
This is an attempt for standardization. We see different solutions for testing everywhere. We also see different naming everywhere. Some solutions even use dependencies (or why is there a `node_modules` ignore rule?). Most coding challenges out there are using a simple I/O approach, we should too. I think that we need to extend our contributor guides and also add some rules there:

- Single file called `solution.$ext` (where `$ext` is the official extension of the programming language).
- No dependencies except those included in the **latest** version of the language. (What about weird languages like C that have super inconsistent standard libraries across systems?)
- No error handling, input is always valid and present.

The last point might be problematic because there are problems for which it would be great to have negative tests to ensure that people actually understood the problem, e.g. #113.

### CODE REVIEW

File: problems/alphanumeric-string-sort/solution.jl [hunk 1]
```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env julia
```
Comment (Fleshgrinder) [inline problems/alphanumeric-string-sort/solution.jl]: New language in the fold. 😎

**Comment by mre** [comment]
Looks great!
I really like that the test environment is language-agnostic similar to Google Code Jam and others.
Shall we add a note to `CONTRIBUTING.md` or the `README.md` that describes the test setup for newcomers?
Also, are you planning to add the test run to CI, perhaps with Github Actions?
This can of course be done in a separate commit.
Keep it up!

**Comment by Fleshgrinder** [comment]
I thought about CI but this could take ages to complete.

**Comment by mre** [comment]
Let's get your changes merged and create an issue to tackle CI later.

**Pull request merged**